canal/canal_test.go

```diff
@@ -89,8 +91,8 @@ func (s *canalTestSuite) execute(c *C, query string, args ...interface{}) *mysql
 
 type testEventHandler struct {
 	DummyEventHandler
-
-	c *C
+	c                *C
+	OnPosSyncedCalls []string
```

Comment:
anywhere to use it?

Reply:
Didnt fully cleanup the previously implemented test case, doing it now